### PR TITLE
test(tgc): cleanup binary from docker image

### PR DESCRIPTION
**Pull request by iyabchen**
test(tgc): cleanup binary from docker image

The ci job and not used varaibles are cleanuped in 
https://github.com/GoogleCloudPlatform/magic-modules/pull/9503

**Release Note Template for Downstream PRs (will be copied)**

```release-note:none

```

### CODE REVIEW

**Comment by ScottSuarez** [comment]
Going to wait until next week to approve. Feel free to ping me

**Comment by melinath** [comment]
@ScottSuarez would we be good to merge this now, if it looks good?

**Review by ScottSuarez — APPROVED** [review]

**Comment by ScottSuarez** [comment]
Still waiting on https://github.com/GoogleCloudPlatform/magic-modules/pull/9503 to be merged.

Sorry does Irys not have ability to merge pull requests? I have presumed they had a authority

**Comment by melinath** [comment]
@iyabchen doesn't have write access for this repository.

**Review by ScottSuarez — APPROVED** [review]

**Pull request merged**
